Formula/heartbeat.rb

```diff
@@ -2,8 +2,8 @@ class Heartbeat < Formula
   desc "Lightweight Shipper for Uptime Monitoring"
   homepage "https://www.elastic.co/products/beats/heartbeat"
```

Comment:
```
==> FAILED
Error: 1 problem in 1 formula detected
heartbeat:
  * The URL https://www.elastic.co/products/beats/heartbeat is not reachable (HTTP status code 404)
```

https://www.elastic.co/products/beats/heartbeat now redirects to https://www.elastic.co/beats/heartbeat, we could try to change it as well:

```suggestion
  homepage "https://www.elastic.co/beats/heartbeat"
```